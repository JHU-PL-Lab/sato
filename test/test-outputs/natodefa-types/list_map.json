[
  {
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [ "n", "hd" ],
          "err_match_val": "true",
          "err_match_expected": "Integer",
          "err_match_actual": "Boolean"
        }
      ]
    ],
    "err_input_seq": [ 1 ],
    "err_location": "n + 1",
    "err_steps": 4935
  },
  {
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [ "n", "hd" ],
          "err_match_val": "true",
          "err_match_expected": "Integer",
          "err_match_actual": "Boolean"
        }
      ]
    ],
    "err_input_seq": [ 0 ],
    "err_location": "n + 1",
    "err_steps": 5929
  }
]