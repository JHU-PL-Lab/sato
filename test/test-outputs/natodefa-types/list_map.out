Type errors for:
- Input sequence : [0]
- Found at expr  : n + 1
--------------------
* Value    : n = hd = true
* Expected : Integer
* Actual   : Boolean
1 errors found. Further control flows may exist.
